npm ci & npm server:start issue

Trying to install the whiteboard server for the first time, following the instructions on the main github page and it fails at the NPM CI instruction.  Don't know how to fix or work around it :(

npm ci
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @chenfengyuan/vue-qrcode@2.0.0
npm ERR! Found: vue@2.7.16
npm ERR! node_modules/vue
npm ERR!   vue@"^2.7.16" from the root project
npm ERR!   peer vue@"^2.0.0" from @cypress/vue2@2.1.1
npm ERR!   node_modules/@cypress/vue2
npm ERR!     dev @cypress/vue2@"^2.1.1" from the root project
npm ERR!   25 more (@linusborg/vue-simple-portal, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer vue@"^3.0.0" from @chenfengyuan/vue-qrcode@2.0.0
npm ERR! node_modules/@chenfengyuan/vue-qrcode
npm ERR!   @chenfengyuan/vue-qrcode@"^2.0.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: vue@3.5.13
npm ERR! node_modules/vue
npm ERR!   peer vue@"^3.0.0" from @chenfengyuan/vue-qrcode@2.0.0
npm ERR!   node_modules/@chenfengyuan/vue-qrcode
npm ERR!     @chenfengyuan/vue-qrcode@"^2.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

&&&

npm ERR! Missing script: "server:start"
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2025-01-08T17_28_13_548Z-debug-0.log

-------

npm run
Lifecycle scripts included in nextcloud@1.0.0:
  test
    jest

available via `npm run-script`:
  build
    webpack --node-env production --progress
  postbuild
    build/npm-post-build.sh
  dev
    webpack --node-env development --progress
  watch
    webpack --node-env development --progress --watch
  lint
    eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then printf "apps/$appdir "; fi; done) core --no-error-on-unmatched-pattern
  lint:fix
    eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then printf "apps/$appdir "; fi; done) core --no-error-on-unmatched-pattern --fix
  test:watch
    jest --watch
  test:coverage
    jest --coverage
  test:jsunit
    karma start tests/karma.config.js --single-run
  sass
    sass --style compressed --load-path core/css core/css/ $(for cssdir in $(find apps -mindepth 2 -maxdepth 2 -name "css"); do if ! $(git check-ignore -q $cssdir); then printf "$cssdir "; fi; done)
  sass:watch
    sass --watch --load-path core/css core/css/ $(for cssdir in $(find apps -mindepth 2 -maxdepth 2 -name "css"); do if ! $(git check-ignore -q $cssdir); then printf "$cssdir "; fi; done)
  sass:icons
    babel-node core/src/icons.js
  cypress
    npm run cypress:component && npm run cypress:e2e
  cypress:component
    cypress run --component
  cypress:e2e
    cypress run --e2e
  cypress:gui
    cypress open
  cypress:version
    cypress version

Are you sure you are running those commands in the whiteboard git repository? ` npm ERR! While resolving: @chenfengyuan/vue-qrcode@2.0.0` this is not a dependency we have in place in this repo.

--- a/composer.json
+++ b/composer.json
@@ -4,7 +4,7 @@
 		"autoloader-suffix": "Whiteboard",
 		"optimize-autoloader": true,
 		"platform": {
-			"php": "8.0"
+			"php": "8.1"
 		},
 		"sort-packages": true
 	},

--- a/lib/Settings/Admin.php
+++ b/lib/Settings/Admin.php
@@ -1,4 +1,5 @@
 <?php
+
 /**
  * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
  * SPDX-License-Identifier: AGPL-3.0-or-later

--- a/lib/Settings/Section.php
+++ b/lib/Settings/Section.php
@@ -1,4 +1,5 @@
 <?php
+
 /**
  * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
  * SPDX-License-Identifier: AGPL-3.0-or-later

--- a/tests/psalm-baseline.xml
+++ b/tests/psalm-baseline.xml
@@ -1,2 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="5.25.0@01a8eb06b9e9cc6cfb6a320bf9fb14331919d505"/>
+<files psalm-version="5.26.1@d747f6500b38ac4f7dfc5edbcae6e4b637d7add0">
+  <file src="lib/Service/ExceptionService.php">
+    <ArgumentTypeCoercion>
+      <code><![CDATA[$statusCode]]></code>
+    </ArgumentTypeCoercion>
+  </file>
+</files>

